spec/unit/crypto.spec.ts

```diff
@@ -167,6 +167,35 @@ describe("Crypto", function () {
 
             client.stopClient();
         });
+
+        it("doesn't throw an error when attempting to decrypt a redacted event", async () => {
```

Comment:
Could we also assess that `redacted_because` is correctly set? That sounds like an important edge to test against